CMakeLists.txt

```diff
@@ -76,6 +76,38 @@ endif()
 include_directories(${GLFW_DIR}/include)
 target_link_libraries(${CMAKE_PROJECT_NAME} glfw ${GLFW_LIBRARIES})
 
+# FMOD
```

Comment:
This is the fmod portion of my old make, not sure if it's cross compatible 
# FMOD
set(FMOD_DIR "$ENV{FMOD_DIR}")
if(NOT FMOD_DIR)
  message(FATAL_ERROR "Failed to link FMOD")
endif()
include_directories(${FMOD_DIR}/inc)
link_directories(${FMOD_DIR}/lib)
target_link_libraries(${CMAKE_PROJECT_NAME} ${FMOD_DIR}/lib/fmod_vc.lib)

Reply:
Me neither. I modeled the cmake code off of the glfw code. I'd like a mac or linux user to try to build with it and see if it works, since I have 0 experience with those platforms.